src/classes/Auth/Auth.ts

```diff
@@ -44,12 +44,17 @@ export class Auth extends PassageBase {
             }
 
             const {
-                payload: { sub: userId },
+                payload: { sub: userId, aud },
             } = await jwtVerify(jwt, this.jwks);
 
             if (!userId) {
                 throw new PassageError('Could not validate auth token.');
             }
+            if (Array.isArray(aud)) {
+                if (!aud.includes(this.config.appId)) {
+                    throw new PassageError('Incorrect app ID claim in token.');
```

Comment:
I know the plan on the next major version is to make this kind of error a normal `Error` and not a `PassageError`. I debated which to do here and went with being consistent with the current choice but I'm not fully sold on this decision. Thoughts?

Reply:
fwiw SonarCloud does not agree with using PassageError